bindings/pyroot/cppyy/CPyCppyy/src/CPPOverload.cxx

```diff
@@ -1072,10 +1072,19 @@ void CPyCppyy::CPPOverload::Set(const std::string& name, std::vector<PyCallable*
     if (name == "__init__")
         fMethodInfo->fFlags |= (CallContext::kIsCreator | CallContext::kIsConstructor);
 
-// special case, in heuristics mode also tag *Clone* methods as creators
-    if (CallContext::sMemoryPolicy == CallContext::kUseHeuristics && \
-            name.find("Clone") != std::string::npos)
-        fMethodInfo->fFlags |= CallContext::kIsCreator;
+    // special case, in heuristics mode also tag *Clone* methods as creators. Only
+    // check that Clone is present in the method name, not in the template argument
+    // list.
+    if (CallContext::sMemoryPolicy == CallContext::kUseHeuristics) {
+       auto name_maybe_template = name;
```

Comment:
Is there a way to search in the substring without copying the string? Algorithmically, that should be possible. Just for the sake of performance

Reply:
I can make it a `string_view`, but a copy needs to happen anyway to pass the right string to the `find` method

Reply:
Well, now you're using `std::string_view::find`, which is what I meant. Sorry for not being clear before!